test/test_bcm/test_bcm.py

```diff
@@ -0,0 +1,118 @@
+from ce_expansion.npdb import db_inter
+from pprint import pprint
+from ce_expansion.atomgraph.bcm import BCModel
+import numpy as np
+import os
+from pprint import pprint
+
+import ase.units as units
+
+this_dir = os.path.dirname(os.path.abspath(__file__))
+
+res = db_inter.get_bimet_result('AuCu', 'icosahedron', 55, n_metal1=22)
+
+atoms = res.atoms_obj
+bond_list = res.nanoparticle.bonds_list
+metal_types = ['Cu', 'Au', "Ag"]
+
+import ase.visualize
+#ase.visualize.view(atoms)
```

Comment:
You should clean up the test code. Remove any imports that you don't and commented lines that are not in use.